control_file_gen.pl

```diff
@@ -175,6 +175,7 @@
            "hstime=s" => \$hstime,
            "advisdir=s" => \$advisdir,
            "scriptdir=s" => \$scriptdir,
+           "nscreen=s" => \$nscreen,
```

Comment:
Super tiny _nit_ but can you align the `=>` with the one above or below?

Reply:
of course

Reply:
Fat arrow now lines up with the one above.